patches/minecraft/net/minecraft/util/Session.java.patch

```diff
@@ -0,0 +1,40 @@
+--- a/net/minecraft/util/Session.java
++++ b/net/minecraft/util/Session.java
+@@ -18,6 +18,8 @@
+    private final String field_148257_b;
+    private final String field_148258_c;
+    private final Session.Type field_152429_d;
++   // Forge: Cache of the local session's GameProfile properties.
++   private com.mojang.authlib.properties.PropertyMap properties;
+ 
+    public Session(String p_i1098_1_, String p_i1098_2_, String p_i1098_3_, String p_i1098_4_) {
+       this.field_74286_b = p_i1098_1_;
+@@ -45,12 +47,27 @@
```

Comment:
Why did you drop most of this patch?

Reply:
This was a patch from very old time (https://github.com/MinecraftForge/MinecraftForge/commit/412952be5b68e985b46c5304ee6a0c1ba8dcad2e) and can no longer be null as Joptsimple marks them as required when accepting the args used to creating the session